pydantic_ai_slim/pydantic_ai/profiles/anthropic.py

```diff
@@ -1,8 +1,53 @@
 from __future__ import annotations as _annotations
 
+from dataclasses import dataclass
+
+from .._json_schema import JsonSchema, JsonSchemaTransformer
 from . import ModelProfile
 
+# TODO update when new models are released that support structured outputs
+# https://docs.claude.com/en/docs/build-with-claude/structured-outputs#example-usage
+models_that_support_json_schema_output = ('claude-sonnet-4-5', 'claude-opus-4-1')
```

Comment:
Constants should always be all-caps, but I'm fine moving this comment and var into the function as well